typing/ctype.ml

```diff
@@ -716,8 +716,7 @@ let rec update_level env level expand ty =
         with Cannot_expand ->
           (* +++ Levels should be restored... *)
           (* Format.printf "update_level: %i < %i@." level (get_level env p); *)
-          if level < get_level env p then raise (Unify [(ty, newvar2 level)]);
```

Comment:
I would rather you do not mix changes to `Ctype` here, as this is completely irrelevant.
As a side note, I'm doing the same change (which seems indeed OK) in #1588, where it is relevant.

Reply:
Done.